src/test/java/org/symphonyoss/symphony/messageml/elements/form/TextAreaTest.java

```diff
@@ -16,13 +16,16 @@
   private static final String NAME_VALUE = "A name";
   private static final String PLACEHOLDER_VALUE = "A placeholder";
   private static final String INITIAL_VALUE = "An initial value";
+  private static final String LINE_BREAK = "An \n initial \n value";
```

Comment:
`LINE_BREAK` could you name this constant another way ? While reading the code, my instinct tells me that this variable equals `\n`. You can maybe name it `INITIAL_VALUE_WITH_LINE_BREAK`

Reply:
sure

Reply:
done